apps/aecore/src/aec_block_generator.erl

```diff
@@ -91,13 +107,13 @@ handle_cast(start_generation, State) ->
     {noreply, do_start_generation(State)};
 handle_cast({worker_done, Pid, {candidate, Candidate, CandidateState}},
             State = #state{ worker = {Pid, _} }) ->
-    %% Only publish non-empty micro-blocks
+    %% Only publish non-empty microblocks
     case aec_blocks:txs(Candidate) of
         [] ->
-            lager:debug("New empty microblock candidate generated, and discarded", []),
+            lager:debug("Empty microblock candidate prepared", []),
```

Comment:
`discarded` was the important bit here, I don't like it being removed from the message.

Reply:
I was trying to find a wording that made sense. It's not discarded, just not published, and stays in the state ready to be filled when there are incoming tx events. Any better suggestion?